data.sql

```diff
@@ -29,4 +29,73 @@ VALUES
     ('Angemon', '12/06/2005', 1, TRUE, -45.0),
     ('Boarmon', '07/06/2005', 7, TRUE, 20.4),
     ('Blossom', '13/10/1998', 3, TRUE, 17.0),
-    ('Ditto', '14/05/2022', 4, TRUE, 22.0);
\ No newline at end of file
+    ('Ditto', '14/05/2022', 4, TRUE, 22.0);
```

Comment:
* Kindly use the right Date format `YYYY-MM-DD` for all your date entries as i get an error while trying to run them [Read up on Date formats here ](https://www.w3schools.com/sql/sql_dates.asp)

Reply:
thanks @Davidosky007 , added the required changes